### ISSUE TITLE
Package new user counts are misleading overcounts.

### ISSUE THREAD

**Issue by LilithHafner** [JuliaComputing/JuliaHub-Feedback#146]
They are labeled "XXX new users in the last 30 days", but seem to be computed as the sum of unique addresses each day. The text indicates that they should be computed as the number of previously unseen users (or IP addresses) over the 30-day span. With the current methodology, a user that downloads a package once per day from the same IP address will be counted as "30 new users in the last 30 days", even if they are not a new user at all.
<img width="363" alt="Screen Shot 2022-07-15 at 9 31 56 AM" src="https://user-images.githubusercontent.com/60898866/179233290-e0295437-0500-4eba-9fe0-ae312ec79d71.png">

**Comment by StefanKarpinski** [comment]
Yeah, you can't just add up the new users per day and get a new users for the month, so that should absolutely be fixed. However, I'm not sure we provide that information in a rollup and I don't particularly want to just add a whole new set of rollups, especially because the way we're generating this data is currently costing us a bunch of JuliaLang money. I also think that the new users stats here are not terribly informative. I suspect that non-CI downloads rank over the past year, aggregated by week would likely be more informative.

**Comment by StefanKarpinski** [comment]
Bottom line:

- yes this is wrong
- there's a bunch of work for me to do here before we do anything to improve this

Thanks for reporting, @LilithHafner
